src/app/page.module.css

```
@@ -27,35 +35,264 @@
+    font-weight: 500;
+    font-size: 3rem;
+    margin-bottom: 3rem;
+    text-align: center;
+    color: #FFFFFF;
+}
+.trendingImageContainer{
+    width: 545px;
+    height: 558px;
+    top: 1310px;
+    left: 100px;
+    radius: 16px;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)